index.js

```diff
@@ -5,7 +5,7 @@ var path = require('path')
 var thesaurus = require('thesaurus')
 var union = require('lodash.union')
 var words = {}
-var firstWordRegex = new RegExp(/^([\w\-]+),/)
+var firstWordRegex = new RegExp(/^([\w-]+),/)
```

Comment:
Thanks for the PR, @gregrperkins! Looks good, just wondering why you changed this? Seems unrelated to the issue.